libexec/base-deploy

```diff
@@ -288,6 +295,7 @@ cat > "$OUT_DIR"/addresses.json <<EOF
     "ILK_REGISTRY": "$ILK_REGISTRY",
     "OSM_MOM": "$OSM_MOM",
     "FLIPPER_MOM": "$FLIPPER_MOM",
+    "MCD_IAM_AUTO_LINE": "$MCD_IAM_AUTO_LINE",
```

Comment:
This doesn't appear to have been added to the 1.2.1 mcd-changelog update. I've added a card to clubhouse.